src/m4a.c

```diff
@@ -0,0 +1,1421 @@
+#include "gba/m4a.h"
+
+
+extern char SoundMainRAM_Buffer[0x380] ;
+
+extern struct SoundInfo gSoundInfo ;
```

Comment:
Why do you need an extra space before `;`? Not consistent with other parts of code